debian source build fails on stock Ubuntu 14.04

At the command `dpkg-buildpackage -S`, the build fails with:

    dpkg-buildpackage: source package walinuxagent
    dpkg-buildpackage: source version 1.3-0ubuntu1
    dpkg-buildpackage: source distribution raring
    dpkg-buildpackage: source changed by Ben Howard <ben.howard@ubuntu.com>
     dpkg-source --before-build WALinuxAgent
    dpkg-checkbuilddeps: Unmet build dependencies: debhelper (>= 8) python-all
    dpkg-buildpackage: warning: build dependencies/conflicts unsatisfied; aborting
    dpkg-buildpackage: warning: (Use -d flag to override.)
    dpkg-buildpackage: warning: this is currently a non-fatal warning with -S, but will probably become fatal in the future
     debian/rules clean
    dh clean --with python2
    make: dh: Command not found
    make: *** [clean] Error 127
    dpkg-buildpackage: error: debian/rules clean gave error exit status 2

This (and subsequent build fail messages) are fixed by installing debhelper, python-all, and python-setuptools.

--- a/debian/README.debian
+++ b/debian/README.debian
@@ -8,7 +8,7 @@ too many extra packages, and keep your source directory clean.
 
 Run once:
     1. Install required packages:
-        sudo apt-get -y install ubuntu-dev-tools pbuilder
+        sudo apt-get -y install ubuntu-dev-tools pbuilder python-all debhelper
     2. Create the pbuilder environment:
         sudo pbuilder create --debootstrapopts --variant=buildd
 

